### PR TITLE
[s] Bump snakeyaml

### PR DESCRIPTION
**Pull request by adamsong**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ dependencies {
     kapt 'org.hibernate:hibernate-jpamodelgen:5.6.7.Final'
     kapt "org.springframework.boot:spring-boot-configuration-processor"
 
+	implementation 'org.yaml:snakeyaml:1.31'
```
Comment (alexkar598) [inline build.gradle]: Why the tab, also what does this do, do we not do dependency pinning?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (adamsong) [reply build.gradle]: IDE was dumb, and there are several packages using snakeyaml, this updates them all to the patched version

Comment (alexkar598) [reply build.gradle]: Is there a way to get a lock file to pin all sub dependencies?

Comment (alexkar598) [reply build.gradle]: This is good enough for merge but iwant to have a discussion about this on discord

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**